src/inputs/radio/radio.jsx

```diff
@@ -9,17 +9,32 @@ import {
 import ClassNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
-import { ENTER_KEY_CODE } from '../../global/constants';
+import {
+    ENTER_KEY_CODE,
+    BEM_RADIO,
+} from '../../global/constants';
 import RadioItem from './radioItem';
 import withStyles from '../../styles/withStyles';
 
 const propTypes = {
+    /**
+     * Aligns the label\'s definition to the left or right.
+     */
     align: PropTypes.oneOf(['left', 'right']),
+    /**
+     * Indicates whether a radio input is checked or not.
+     */
     checked: PropTypes.oneOfType([
         PropTypes.bool,
         PropTypes.number,
     ]),
+    /**
+     * Children are used for pill mode
+     */
     children: PropTypes.node,
+    /**
+     * classes
```

Comment:
Change comment to: "Override or extend the styles applied to Input."

Reply:
done!